### PR TITLE
Remove unnecessary buttons if there are no comments.  

### PR DESCRIPTION
**Pull request by AnudeepYakkala**
Added a function that hides the clear button and the drop-down to select the number comments to display if there are 0 comments stored in Datastore.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -90,13 +90,25 @@ function cancelRedirect() {  // eslint-disable-line no-unused-vars
...
   const numComments = $('#num-comments-selector :selected').val();
   fetch('/data?max-comments=' + numComments)
       .then((response) => response.json())
       .then((messages) => {
+        if (messages.length == 0) {
+          document.getElementById('num-comments-selector').style.display =
+              'none';
+          document.getElementById('clear-button').style.display = 'none';
+        } else {
+          document.getElementById('num-comments-selector').style.display =
```
Comment (chen-dawn) [inline portfolio/src/main/webapp/script.js:107]: display = `block` or `inline`?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -90,13 +90,25 @@ function cancelRedirect() {  // eslint-disable-line no-unused-vars
...
+ * section on the home page. Removes the clear button and the
+ * option to selected number of comments displayed if there are
+ * no comments.
  */
 function getComments() {  // eslint-disable-line no-unused-vars
   const numComments = $('#num-comments-selector :selected').val();
   fetch('/data?max-comments=' + numComments)
       .then((response) => response.json())
       .then((messages) => {
+        if (messages.length == 0) {
```
Comment (chen-dawn) [inline portfolio/src/main/webapp/script.js:102]: if (!messages || messages.length === 0)

messages might be null or undefined

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (AnudeepYakkala) [reply portfolio/src/main/webapp/script.js:107]: Okay, I will change it to inline.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (AnudeepYakkala) [reply portfolio/src/main/webapp/script.js]: Oh, okay.

**Pull request merged**